app/src/main/java/org/greenstand/android/TreeTracker/activities/TreeTrackerActivity.kt

```diff
@@ -15,6 +15,7 @@ class TreeTrackerActivity : ComponentActivity() {
     private val languageSwitcher: LanguageSwitcher by inject()
     private val viewModelFactory: TreeTrackerViewModelFactory by inject()
 
+    //hello this is the first change i have made
```

Comment:
No need to add this comment. Github automatically the lines and file you made changes to.

Reply:
Yeah this was on the first commit, i was trying learn how deal with github.